src/domain/enrolment/CreateEnrolmentPage.tsx

```diff
@@ -0,0 +1,61 @@
+import { Button, IconArrowLeft } from 'hds-react';
+import { useRouter } from 'next/router';
+import React, { ReactElement } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import LoadingSpinner from '../../common/components/loadingSpinner/LoadingSpinner';
+import { useEventQuery } from '../../generated/graphql';
+import { Router } from '../../i18n';
+import Container from '../app/layout/Container';
+import PageWrapper from '../app/layout/PageWrapper';
+import { ROUTES } from '../app/routes/constants';
+import NotFoundPage from '../notFoundPage/NotFoundPage';
+import styles from './enrolmentPage.module.scss';
+import EventInfo from './eventInfo/EventInfo';
+
+const CreateEnrolmentPage = (): ReactElement => {
```

Comment:
Would React.FC be more convenient in component definitions?
const CreateEnrolmentPage: React.FC = () => { ?

Reply:
Fixed